cmake/std/atdm/waterman/environment.sh

```diff
@@ -206,7 +206,6 @@ export ATDM_CONFIG_MPI_POST_FLAGS="-map-by;socket:PE=4"
 
 export ATDM_CONFIG_WORKSPACE_BASE_DEFAULT=/home/atdm-devops-admin/jenkins
 export ATDM_CONFIG_TRIL_CMAKE_INSTALL_PREFIX_DATE_BASE_DEFAULT=/home/atdm-devops-admin/trilinos_installs
```

Comment:
This could be removed.

Reply:
There is no such thing as `/projects/atdm_devops/` on 'waterman' and they were not willing to set one up.  Therefore, we had to use `/home/atdm-devops-admin/` to do Trilinos installs under.